assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -104,6 +104,7 @@ data "template_file" "controller-configs" {
     ssh_keys              = jsonencode(var.ssh_keys)
     k8s_dns_service_ip    = cidrhost(var.service_cidr, 10)
     cluster_domain_suffix = var.cluster_domain_suffix
+    controllers           = var.controller_count
```

Comment:
`controllers_nr` or  `controllers_count` seems more clear, IMHO

Reply:
Maybe `controller_count` to keep it consistent with Terraform?

Reply:
Renamed.